stack.yaml

```diff
@@ -3,6 +3,6 @@
 # * https://github.com/commercialhaskell/stackage/issues/4731
 resolver: nightly-2019-08-07
 packages: [.]
-extra-deps: [ghc-lib-parser-8.8.1]
+extra-deps: [ghc-lib-parser-8.8.1.20191204.tar.gz]
```

Comment:
Why the .tar.gz at the end?

Reply:
oops, my fault. That was leftover from testing. I removed it.